src/components/Info/Info.module.scss

```diff
@@ -0,0 +1,18 @@
+$font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
if you want to use it in multiple files you can also make a vanilla css variable in the global file, and that way you don't have to define it multiple times

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties